galaxy.yml

```diff
@@ -2,6 +2,7 @@
 
 namespace: cloud
 name: common
+version: 2.1.4
```

Comment:
There shouldn't be the need to add version here.

Reply:
version is a required key in galaxy.yaml according to ansible-lint.

Reply:
It should be 2.1.3 then. As far as I know it should match the latest release, but maybe I’m wrong.